packages/@angular/cli/lib/config/schema.json

```diff
@@ -105,6 +105,10 @@
             "type": "string",
             "description": "The name of the TypeScript configuration file for unit tests."
           },
+          "karma": {
```

Comment:
Hm... we could accept both forms (string and object), but at the end of the day it wouldn't be the same as the toplevel one because of the extra test filename, so at best it would be something like:

```
  "test": {
    "input": "test.ts",
    "karma": {
      "config": "./karma.conf.js"
    }
  },
```

Another option would be to have a new key like `testConfig` instead. Either way I don't think it should just be `karma` because it does not mirror the toplevel karma structure (`karma.config`).